src/NormalizingFlows.jl

```diff
@@ -1,5 +1,72 @@
 module NormalizingFlows
 
-# Write your package code here.
+using Bijectors
+using Optimisers
+using LinearAlgebra, Random, Distributions, StatsBase
+using ProgressMeter
+using ADTypes, DiffResults
+using Zygote, ForwardDiff, ReverseDiff, Enzyme
+
+using DocStringExtensions
+
+export NF, elbo, loglikelihood
+
+"""
+    NF(rng::AbstractRNG, vo, flow, args...; kwargs...)
+    NF(vo, flow, args...; kwargs...) = NF(Random.default_rng(), vo, flow, args...; kwargs...)
+
+Train the given normalizing flow `flow` by calling `train`.
+
+# Arguments
+- `rng::AbstractRNG`: random number generator
+- `vo`: variational objective
+- `flow`: normalizing flow to be trained
+- `args...`: additional arguments for `vo`
+- `max_iters::Int=1000`: maximum number of iterations
+- `optimiser::Optimisers.AbstractRule=Optimisers.ADAM()`: optimiser to compute the steps
+- `ADbackend::ADTypes.AbstractADType=ADTypes.AutoZygote()`: automatic differentiation backend
+- `kwargs...`: additional keyword arguments for `train` (See `train`)
```

Comment:
Seperate into `# Arguments` and `# Keyword arguments`; from the above it's not clear to me that `max_iters` is actually a kwarg.